addons/api/addon/models/credential.js

```diff
@@ -3,7 +3,7 @@ import GeneratedCredentialModel from '../generated/models/credential';
 /**
  * Supported Credential types.
  */
-export const types = ['username_password', 'ssh_private_key'];
+export const types = ['username_password', 'ssh_private_key', 'json'];
```

Comment:
FYI, we need to add a feature flag check along with this change. We want to make sure in a production env we cannot access JSON credentials still. I would look at how [credential store abilities](https://github.com/hashicorp/boundary-ui/blob/main/addons/api/addon/abilities/credential-store.js#L18) is using the feature service and do something similar in the [credential ability](https://github.com/hashicorp/boundary-ui/blob/main/addons/api/addon/abilities/credential.js#L12-L14) `canRead`

Reply:
Thank you for catching that! I will look into that.